site/cart.js

```diff
@@ -8,7 +8,7 @@ async function load() {
     let cart = null;
     const cartName = getQueryParameter("name");
     if (cartName) {
-        for (c of carts) {
+        for (c of shoppingCart.carts) {
```

Comment:
Please rename `shoppingCart` to `shoppingCarts` as the instance manages multiple carts.

Similarly, please also rename the class `ShoppingCart` to `ShoppingCarts`

Reply:
Changed class name and variable instance.